faucet/Cargo.toml

```diff
@@ -0,0 +1,23 @@
+[package]
+name = "miden-node-faucet"
+version = "0.1.0"
+edition = "2021"
+
+# See more keys and their definitions at https://doc.rust-lang.org/cargo/reference/manifest.html
+
+[dependencies]
+actix-web = "4"
+actix-files = "0.6.5"
+actix-cors = "0.7.0"
+derive_more = "0.99.17"
+figment = { version = "0.10", features = ["toml", "env"] }
+miden-lib = { workspace = true }
+miden-client = { version = "0.1.0", features = ["testing","concurrent"] }
```

Comment:
Should the `testing` be optional for the client? If the node is deployed without it, the client would not work correctly (although this probably won't be needed anytime soon?)